packages/clay-color-picker/src/index.tsx

```diff
@@ -142,107 +143,128 @@ const ClayColorPicker: React.FunctionComponent<IProps> = ({
-						<Splotch
-							aria-label={ariaLabels.selectColor}
-							className="dropdown-toggle"
-							onClick={() =>
-								useNative && valueInputRef.current
-									? valueInputRef.current.click()
-									: setActive((val: boolean) => !val)
-							}
-							size={28}
-							value={value}
-						/>
-					</ClayInput.GroupText>
-				</ClayInput.GroupItem>
-
-				{showHex && (
-					<ClayInput.GroupItem append>
-						<ClayInput
-							aria-label={sub(ariaLabels.selectionIs, [value])}
-							insetBefore
-							onBlur={event => {
-								const newColor = tinycolor(event.target.value);
-
-								const hexString = newColor.isValid()
-									? newColor.toHex()
-									: value;
-
-								onValueChange(hexString);
-								setHexInputValue(hexString);
-							}}
-							onChange={event => {
-								const newHexValue = event.target.value;
-
-								const newColor = tinycolor(newHexValue);
-
-								setHexInputValue(newHexValue);
-
-								if (newColor.isValid()) {
-									onValueChange(newColor.toHex());
-								}
-							}}
-							ref={inputRef}
-							type="text"
-							value={hexInputValue.toUpperCase().substring(0, 6)}
-						/>
-
-						<ClayInput.GroupInsetItem before tag="label">
-							{'#'}
-						</ClayInput.GroupInsetItem>
-					</ClayInput.GroupItem>
-				)}
-			</ClayInput.Group>
-
-			<DropDown.Menu
-				active={active}
-				alignElementRef={triggerElementRef}
-				className="clay-color-dropdown-menu"
-				onSetActive={setActive}
-				ref={dropdownContainerRef}
-			>
-				{!onColorsChange && (
-					<Basic
-						colors={colors || DEFAULT_COLORS}
-						label={label}
-						onChange={onValueChange}
-					/>
-				)}
-
-				{onColorsChange && (
-					<Custom
-						colors={
-							colors
-								? colors.concat(BLANK_COLORS).slice(0, 12)
-								: BLANK_COLORS
+		<FocusScope arrowKeysUpDown={false}>
+			<div className="clay-color-picker">
+				{name && (
+					<input
+						name={name}
+						onChange={e =>
+							useNative ? onValueChange(e.target.value) : null
 						}
-						label={label}
-						onChange={onValueChange}
-						onColorsChange={onColorsChange}
-						spritemap={spritemap}
+						ref={valueInputRef}
+						style={{display: 'none'}}
+						type={useNative ? 'color' : 'text'}
+						value={value}
 					/>
 				)}
-			</DropDown.Menu>
-		</div>
+
+				{title && <label>{title}</label>}
+
+				<ClayInput.Group className="clay-color" ref={triggerElementRef}>
+					<ClayInput.GroupItem prepend={showHex} shrink>
+						<ClayInput.GroupText>
+							<Splotch
+								aria-label={ariaLabels.selectColor}
+								className="dropdown-toggle"
+								onClick={() =>
+									useNative && valueInputRef.current
+										? valueInputRef.current.click()
+										: setActive((val: boolean) => !val)
+								}
+								ref={splotchRef}
+								size={28}
+								type="button"
+								value={value}
+							/>
+						</ClayInput.GroupText>
+					</ClayInput.GroupItem>
+
+					<DropDown.Menu
```

Comment:
I ended up moving `DropDown.Menu` into the Group just with the intention of having the components before the Input component so that `FocusScope` respects the order of focus and follows the Marcos guidelines. I think it's not a problem since it is always rendered with the Portal.